pytype/pyc/pyc.py

```diff
@@ -76,6 +77,8 @@ def compile_src_string_to_pyc_string(src, filename, python_version, python_exe,
     # We pass -E to ignore the environment so that PYTHONPATH and sitecustomize
     # on some people's systems don't mess with the interpreter.
     cmd = exe + ["-E", "-", fi.name, filename or fi.name, mode]
+    if sys.platform == 'win32':
+      cmd = ["python","-E", "-", fi.name, filename or fi.name, mode]
```

Comment:
i think this would not work for analysing python code of a different version than the host python, right? we should add a comment saying so, and maybe something in the readme about this being a limitation under windows unless the python_exe flag is specified.

Reply:
Thank you @martindemello 

I understand your comment like the following.　Please correct me if I’m wrong.

・Points that should be revised 
This code can not know the difference between python2.x and python3.x.
 
・Your proposal
Write to readme about this being a limitation under windows unless the python_exe flag is specified.
→I think that it is an amazing idea,
  but I think about WindowsPytypeUser must provide a value for python_exe.

・My proposal
I would like to know the difference between python2.x and python3.x.

(An example is  python3.7) 
not working under windows.
`subprocess.check_output("python3.7")`

working under windows.
`subprocess.check_output("py -3.7")`

I would like to use py instead of python.
 
What are your thoughts on that?

Reply:
I think using `py -3.7` is a good idea, although it looks like `py` might not always be available: https://docs.python.org/3/using/windows.html#launcher.

Reply:
one further comment - move the `if win32` check above the `cmd = [...]`, and only change the value of `exe`, since the rest is the same. so something like

````
if win32:
  exe = "py -" + version
else:
  exe = "python" + version
````

Reply:
Thank you @rchen152 @martindemello 

@rchen152 
Thank you for the information. 
I think that I have two choices, what do you think?

1.Add Except 
I'm still thinking about error message.
```
try:
    subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
except FileNotFoundError as e:
    logging.error("FileNotFoundError error 'py' is not recognized: %s\n", str(e))
 ```
2.I will write to readme about "User need check that the launcher is available under windows".

@martindemello 
I  rewrote my code.

Reply:
I think we should do both of those things: write in the README that in Windows, pytype requires Python 3.3+ to analyze a different Python version from the one it's running under, and also report an error in the code when the launcher isn't found.

Reply:
Thanks
I add Except and modify Readme.